### ISSUE TITLE
Enabled true/false vs 0/1

### ISSUE THREAD

**Issue by ZoneMR** [ebaauw/homebridge-hue#66]
When setting the enabled status for a motion sensor via Elgato Eve, I get this log and everything works fine:

`Bathroom Motion Sensor: homekit enabled changed from 0 to 1`

However, when creating a scene that is intended to enable a sensor, I get these log lines and the state immediately resets back to disabled:

```
Bathroom Motion Sensor: homekit enabled changed from 0 to true
Bathroom Motion Sensor: set homekit enabled from true to 0
```

I'm not sure why in one case the request is sent as a boolean, and in the other it's sent as a numeric value, but it seems homebridge-hue should be modified to support both?

**Comment by ebaauw** [comment]
I agree and I'll include the fix in the next release.

**Comment by ebaauw** [comment]
Fixed in v0.3.4.
